### PR TITLE
Update SMPP concept doc

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Draft SMPP concept doc after discussion with @thepeachbeetle and @parthawasthi. 

## Deploy Notes

Check `Documentation > Messaging > SMS > Guides > SMPP access` in the [review app](https://nexmo-developer-pr-1199.herokuapp.com/): 
Marking as `don't merge` so that Paul and Parth can review.

### CODE REVIEW

File: _documentation/messaging/sms/guides/SMPP-access.md [hunk 1]
```diff
@@ -41,14 +71,32 @@ To use our configuration:
...
 
-> Note: Check out our [SMPP FAQ] for information about supported character encodings, delivery receipt formats, throttle management, concatenated messages and more.
+## Our SMPP instances
+
+Nexmo hosts three instances of SMPP:
+
+- `SMPP1/2` - clustered
+- `SMPP0` - not clustered
+
+We recommend that you bind to `SMPP1/2`.
```
Comment (parthawasthi) [inline _documentation/messaging/sms/guides/SMPP-access.md]: We recommend that you bind to both these clusters for ensuring the best possible redundancy. If not feasible, please bind to either one of the two clusters.

Comment (marklewin) [reply _documentation/messaging/sms/guides/SMPP-access.md]: Just to clarify: my understanding is that `SMPP1/2` is a cluster and `SMPP0` is not? If so, we can't really say "both of these clusters". The [KB article](https://nexmoinc.atlassian.net/wiki/spaces/MSG/pages/590086408/SMPP+Availability+Internal+Communication) suggests that they should not bind to `SMPP0` at all unless they cannot bind to `SMPP1/2`. Is that incorrect?

**Comment by marklewin** [comment]
Thanks very much @parthawasthi. I am adding @lornajane for a general review after Paul has had a chance to look at it. If you could please check my query about your earlier comment I'd appreciate it.

File: _documentation/messaging/sms/guides/SMPP-access.md (hunk 1 above)
Comment (parthawasthi) [reply _documentation/messaging/sms/guides/SMPP-access.md]: That is correct, Mark and your choice of words is accurate and more meaningful. What I mean is, we advise customers to bind to (the cluster) SMPP1 and in the event their bind to it goes down, initiate bind with (the cluster) SMPP2; not necessarily in that order. That way even if we lost a whole cluster (SMPP1 or 2) they will not be impacted. If they don't want to (or cannnot) do this, they must bind to either SMPP1 or 2, both of which are clustered, and thereby offer redundancy.

Comment (thepeachbeetle) [reply _documentation/messaging/sms/guides/SMPP-access.md]: so to clarify, the 'cluster' means  both smpp1 and smpp2 (i.e., they are each 1 half of the cluster),  to be bound to the cluster you should be bound to both smpp1 and smpp2 at all times (and if you are not able to do that, then smtp0 is more appropriate for you (which isn't a cluster at all but is a single standalone endpoint

**Comment by marklewin** [comment]
Thanks @parthawasthi and @thepeachbeetle. I have updated the doc to recommend binding to both `SMPP1` **and** `SMPP2` and only binding to `SMPP0` if you cannot connect to a clustered setup.

**Pull request merged**